libafl/src/executors/command.rs

```diff
@@ -494,6 +493,76 @@ where
     }
 }
 
+/// The shared implementation for children with stdout/stderr/timeouts.
+pub trait ChildrenArgs: Sized {
+    /// The timeout of the children
+    fn timeout_ref(&self) -> &Duration;
+    /// The mutable refernce of the timeout of the children
+    fn timeout_mut(&mut self) -> &mut Duration;
+
+    /// The stderr handle of the children
+    fn stderr_ref(&self) -> &Option<Handle<StdErrObserver>>;
+    /// The mutable stderr handle of the children
+    fn stderr_mut(&mut self) -> &mut Option<Handle<StdErrObserver>>;
+
+    /// The stdout handle of the children
+    fn stdout_ref(&self) -> &Option<Handle<StdOutObserver>>;
+    /// The mutable stdout handle of the children
+    fn stdout_mut(&mut self) -> &mut Option<Handle<StdOutObserver>>;
+
+    /// The current directory of the spawned children
+    fn current_dir_ref(&self) -> &Option<PathBuf>;
+    /// The mutable current directory of the spanwed children
+    fn current_dir_mut(&mut self) -> &mut Option<PathBuf>;
+
+    /// Whether debug child by inheriting stdout/stderr
+    fn debug_child_ref(&self) -> &bool;
+    /// Whether debug child by inheriting stdout/stderr
+    fn debug_child_mut(&mut self) -> &mut bool;
+
+    // --- Builder methods ----
+
+    #[must_use]
+    /// Sets the execution timeout duration.
+    fn timeout(mut self, timeout: Duration) -> Self {
```

Comment:
`set_timeout`

Reply:
This is the original API name and it is the builder pattern.